<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropScene3D</class>
 <widget class="QWidget" name="PropScene3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>147</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>147</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PropScene3D</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Ambient light:</string>
       </property>
       <property name="buddy">
        <cstring>AmbientColor</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Armed::ColorButton" name="AmbientColor">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Fog type:</string>
       </property>
       <property name="buddy">
        <cstring>FogType</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="FogType"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelFogColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Color:</string>
       </property>
       <property name="buddy">
        <cstring>FogColor</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Armed::ColorButton" name="FogColor">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LabelFogDensity">
       <property name="text">
        <string>&amp;Density:</string>
       </property>
       <property name="buddy">
        <cstring>FogDensity</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="FogDensity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="LabelFogStart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Start:</string>
         </property>
         <property name="buddy">
          <cstring>FogStart</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="FogStart"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="LabelFogEnd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;End:</string>
         </property>
         <property name="buddy">
          <cstring>FogEnd</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="FogEnd"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Armed::ColorButton</class>
   <extends>QPushButton</extends>
   <header>ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
